ios/Codeforces Watcher/Application/features/contests/ContestsNavigationBar.swift

```diff
@@ -93,6 +74,22 @@ struct ContestsNavigationBar: View {
             }
         }
     }
+    
+    private func RightButton(_ imageName: String) -> some View {
```

Comment:
Probably, `ToggleButton` would be less specific and more universal at the same time.